narwhals/_pandas_like/utils.py

```diff
@@ -270,12 +270,17 @@ def reuse_series_namespace_implementation(
     )
 
 
-def item(s: Any) -> Any:
+def item(s: Any, index: int | None = None) -> Any:
```

Comment:
So.. I didn't want to move it directly to the pandas_like series because I noticed it is used in grouped object as well, which I am not confident in touching at almost midnight.

Implementation also follows polars one

Reply:
Edit: Moved it to series directly, nothing breaks in tests 👌